### ISSUE TITLE
Importing svgbob as wasm within an html file

### ISSUE THREAD

**Issue by ramu-r** [ivanceras/svgbob#63]
I compiled svgbob using `wasm-pack build --target web` and I'm trying to import it into an html file as described [here](https://rustwasm.github.io/wasm-bindgen/examples/without-a-bundler.html) but it gives me import errors and I don't know what I'm supposed to import. I tried importing both svgbob and to_svg neither of which worked.

Can you give me any pointers on how to use this within an html file?

**Comment by ivanceras** [comment]
What does the error says?

**Comment by ramu-r** [comment]
The error says `Uncaught SyntaxError: import not found: svgbob`
I tried importing some other things too and none of them worked.

The html file basically contains 

```
    <script type="module">
      import init, { svgbob } from './svgbob.js';

      async function run() {
        const result = svgbob.to_svg("------------->");
        document.write(result);
        console.log(`${result}`);
      }

      run();
    </script>
```

My understanding of rust is pretty non existent and I wouldn't know if I'm doing something wrong here. I also found [svgbob-wasm](https://github.com/agoose77/svgbob-wasm/) which seems to get the job done but uses an older version of svgbob.

So if svgbob wasn't meant to do this kinda thing, this ticket can be closed. But if this is something easily solvable that I'm overlooking, I'd like to try compiling svgbob to wasm and use it.
